test/src/commonMain/kotlin/test/Nullability.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import io.realm.runtimeapi.RealmModel
+import io.realm.runtimeapi.RealmObject
+
+@RealmObject
+class Nullability : RealmModel {
```

Comment:
can we extend to other supported types as well? (Long, Byte etc.)

Reply:
We could. I was a bit in doubt on how to define the axis for these basic test model/cases. The idea here was to define a specific class for the nullability test, to avoid the `AllTypes` as in Java, which is heavily confusing, because it has so many uses. Should I just include all non-containerized types here and verify them all? Or should we try to build the AllTypes++?

Reply:
Let's figure this out in the next Sprint

Reply:
Yep. I created #133 for coming up with an appropriate pattern for exhaustive test that also breaks when we add new types.